src/activities/assignments/AssignmentEntity.js

```diff
@@ -258,4 +258,28 @@ export class AssignmentEntity extends Entity {
 		];
 		await performSirenAction(this._token, action, fields);
 	}
+
+	annotationToolsAvailable() {
+		return this._entity.getSubEntityByClasses(['annotations', 'enabled']);
+	}
+
+	canSeeAnnotations() {
```

Comment:
Adding a quick JSDoc to each of these would be appreciated! We're not perfect on it, but all new things should have them.